src/toast/ops/sim_ground.py

```diff
@@ -237,10 +239,56 @@ def _check_schedule(self, proposal):
                 )
         return sch
 
+    # Cross-check HWP parameters
```

Comment:
Just a note:  when validating a set of traits that are inter-dependent like this, we can get into a situation where none of the traits can be changed, or where an error can occur if the user does not set traits in a particular order.  For example, if all 3 traits start as `None`, then we can't set `hwp_angle` (since the other traits are None) and we can't set `hwp_rpm`, because `hwp_angle` is None.  The work arounds are:
1.  Don't use the traitlets validate for these kinds of inter-connected traits.  Instead, just check consistency at the beginning of the `_exec()` method when they are about to be used.
2.  Set all the traits inside a `hold_trait_notifications` context: 
 https://traitlets.readthedocs.io/en/stable/using_traitlets.html#basic-example-validating-the-parity-of-a-trait

So for example:
```
sim_ground = ops.SimGround()
with sim_ground.hold_trait_notifications():
    sim_ground.hwp_rpm = 5.0
    sim_ground.hwp_angle = "hwp_angle"
```
Either way is fine, we should just make sure to note this when overhauling the documentation and discussing Operator traits.